LuaUI/Widgets/gui_chili_share.lua

```diff
@@ -54,8 +55,10 @@ local images = {
 	giftenergy = 'LuaUI/Images/energy.png',
 }
 local defaultamount = 100
+local zeroIfNumWrhDisabled = 0
```

Comment:
Weird name. Also consider a proper type and apply logic later:
```suggestion
local whrEnabled = false
```
and then later:
```diff
- stuff + zeroIfNumWrhDisabled * whrWidth
+ stuff + (whrEnabled and whrWidth or 0)
```

Reply:
Was trying to keep the usage of Lua's somewhat dangerous replacement for the ternary operator in 1 place (actually 2), but ok :)